readme.md

```diff
@@ -1,14 +1,138 @@
 # Sequelize Quick SSCCE Base Repository
 
-Use this repository to quickly create an [SSCCE](http://www.sscce.org/) for your issue! This way your issue will be much easier to investigate.
+Use this repository to create an [SSCCE](http://www.sscce.org/) for your issue! It will greatly help us figure out what is going wrong and your issue
+will be much easier to investigate.
 
-By using this repository, you won't have to worry about setting up any database. You don't need to install anything, you don't need docker, you don't need to spend time configuring a development environment to create your SSCCE. Everything is already set up for you.
+## Method 1: Create your SSCCE locally (preferred)
 
-**You just [write your code](src/sscce.js) and it works, directly from GitHub!**
+### Step 1 - Install this repository locally
 
-## Step 1 - Create the SSCCE
+Start by [Forking this repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo),
+then [clone it on your machine](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository).
 
-Go to the [src/sscce.js](https://github.com/papb/sequelize-sscce/blob/master/src/sscce.js) file in this repository, and click the edit button (a pencil).
+Run `npm install` or `yarn` to install the necessary dependencies.
+
+### Step 1 - Create the SSCCE
+
+Modify [src/sscce.ts](./src/sscce.ts) to create a failing test that accurately represents the issue you're experiencing.
+
+Remember the [SSCCE rules](http://www.sscce.org/). It should be:
+
+- *Short (Small)* - It should not include anything that is not relevant to your issue.
+- *Self Contained* - Ensure everything is included, ready to go.
+- *Correct* - It should demonstrate the problem you're encountering (i.e. the sscce should fail, but with the right error).
+- *Example* - Displays the problem we are trying to solve.
+
+### Step 2 - Run the SSCCE
+
+Use one of our npm scripts to run your SSCCE.
+
+Running with a dialect other than sqlite will require installing an extra package
+& having a database running. For convenience, we provide docker containers for each database.
+
+You can also opt to open a pull request, and run the code directly in our CI.
+
+#### sqlite
+
+The easiest of the bunch. Simply run:
+
+```shell
+npm run test:sqlite
+```
+
+#### postgres
+
+You'll need to install the `pg` package and have a postgres database running.
```

Comment:
pg-hstore is not needed?

Reply:
It's only needed if you use DataTypes.HSTORE, I'll document it